apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1797,8 +1797,8 @@
                 "protocol_beneficiaries" : {
                     "type" : "array",
                     "items" : {
-                        "description" : "Public keys belonging to protocol maintainers with reward shares (100 is 10%). If not set testnet or mainnet beneficiaries and shares will be used based on network_id configuration value. IMPORTANT: The value of this setting is under governance, thus it should not be changed without previous agreement within the configured network on doing so.",
-                        "pattern": "^ak_[1-9A-HJ-NP-Za-km-z]*:[0-9]+$"
+                        "description" : "Public keys belonging to protocol maintainers with reward shares (100 is 10%), optionally a protocol version can be specified from and/or to this account will be rewarded, e.g. ak_2KAcA2Pp1nrR8Wkt3FtCkReGzAi8vJ9Snxa4PcmrthVx8AhPe8:109:6: specifies from Ceres onwards, ak_2KAcA2Pp1nrR8Wkt3FtCkReGzAi8vJ9Snxa4PcmrthVx8AhPe8:109::5 specifies to Iris. If not set testnet or mainnet beneficiaries and shares will be used based on network_id configuration value. IMPORTANT: The value of this setting is under governance, thus it should not be changed without previous agreement within the configured network on doing so.",
```

Comment:
The protocol_beneficiaries is a list of beneficiaries rather than a single beneficiary. If it was a single beneficiary then it would be implicit but with a list I don't think it is.

Reply:
Yes, Addr:BRIAmount:: is the common case but if you don't need specify the to and from in this case, they are optional. You could just specify Addr:BRIAmount

From the code, it looks like it is possible to specify multi BRI accounts, it adds them together and makes sure they don't go above a certain limit.

Reply:
okay, if there is technical requirement that's fine

Reply:
Calling it a requirement is perhaps a bit strong, but I think this variant is slightly more fool-proof. Though I'm sure Nikita or Thomas would break it in a minute 😅

Reply:
This is how a fool would try to change the beneficiary
1. Add NewAddr:1000 to config 
That fails, because the old beneficiary is there and now the total is more than 100% ???
2.  remove Addr:1000 from config.... 
That fails, I guess, because synchronizing and validating old blocks would no longer work?
Possibly not something one would notice straight away, but as soon as one starts that config on a new node to sync??
3. Introduce a new protocol vsn 2 and then configure
Addr:1000::1
NewAddr:1000::
Because someone said that that was the default use-case.
But it is actually better to say
NewAddr:1000:2:

Did I understand the documentation well?